recipe/0001-Fix-build-on-ppc.patch

```diff
@@ -0,0 +1,36 @@
+From c92c7f04f6e7cc2f7adc4ec4557c255a8e26ade3 Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Wed, 31 Mar 2021 08:58:25 +0000
+Subject: [PATCH] Fix build on ppc
+
+---
+ configure.py | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/configure.py b/configure.py
+index 4772a55..826a78d 100644
+--- a/configure.py
++++ b/configure.py
+@@ -48,6 +48,10 @@ def is_linux():
+     return platform.system() == "Linux"
+ 
+ 
++def is_ppc64le():
++    return os.uname()[4] == "ppc64le"
```

Comment:
platform.machine() works as well

Reply:
This is fine for this patch. We should fix this upstream, however, and give it `is_linux_x86_64` ,  `is_linux_ppc64le` and `is_linux_aarch64` functions in case there are specific options we need for each.

Reply:
Yeah. I agree.

Reply:
https://github.com/tensorflow/addons/pull/2438